nomad/structs/diff_test.go

```diff
@@ -2805,6 +2805,12 @@ func TestTaskGroupDiff(t *testing.T) {
 										Old:  "",
 										New:  "",
 									},
+									{
+										Type: DiffTypeNone,
+										Name: "Body",
+										Old:  "",
```

Comment:
While this _should_ be trivial and rightfully not worth testing, but unfortunately it's really easy for us to break diffing. 😊 So let's add a `Body` to this test.

Reply:
No worries! I've updated the test to check for this now